ocb3/src/lib.rs

```diff
@@ -77,7 +77,21 @@ mod private {
     impl SealedTagSize for consts::U16 {}
 
     // Nonces are <= 120 bits
+    impl SealedNonceSize for consts::U1 {}
+    impl SealedNonceSize for consts::U2 {}
+    impl SealedNonceSize for consts::U3 {}
+    impl SealedNonceSize for consts::U4 {}
+    impl SealedNonceSize for consts::U5 {}
+    impl SealedNonceSize for consts::U6 {}
+    impl SealedNonceSize for consts::U7 {}
+    impl SealedNonceSize for consts::U8 {}
+    impl SealedNonceSize for consts::U9 {}
+    impl SealedNonceSize for consts::U10 {}
+    impl SealedNonceSize for consts::U11 {}
     impl SealedNonceSize for consts::U12 {}
+    impl SealedNonceSize for consts::U13 {}
+    impl SealedNonceSize for consts::U14 {}
+    impl SealedNonceSize for consts::U15 {}
```

Comment:
Hmm, should we change this to an [`LeEq`](https://docs.rs/typenum/latest/typenum/operator_aliases/type.LeEq.html) constraint instead?

Reply:
The same probably goes for `TagSize`

Reply:
yes that seems better

Reply:
okay, I have no idea how to use this though to make a blanket impl 😅

Reply:
I opened #590 to use `IsLessOrEqual` for both `NonceSize` and `TagSize`

Reply:
rebased accordingly